metadata-ingestion/examples/mce_files/bootstrap_mce.json

```diff
@@ -2916,217 +2931,222 @@
-                    "source": null
+            }
+          },
+          {
+            "com.linkedin.pegasus2avro.dataset.UpstreamLineage": {
+              "upstreams": [
+                {
+                  "auditStamp": {
+                    "time": 1581407189000,
+                    "actor": "urn:li:corpuser:jdoe",
+                    "impersonator": null
                   },
-                  {
-                    "owner": "urn:li:corpuser:datahub",
-                    "type": "DATAOWNER",
-                    "source": null
-                  }
-                ],
-                "lastModified": {
-                  "time": 1581407189000,
-                  "actor": "urn:li:corpuser:jdoe",
-                  "impersonator": null
+                  "dataset": "urn:li:dataset:(urn:li:dataPlatform:hive,logging_events,PROD)",
+                  "type": "COPY"
                 }
-              }
-            },
-            {
-              "com.linkedin.pegasus2avro.dataset.UpstreamLineage": {
-                "upstreams": [
-                  {
-                    "auditStamp": {
-                      "time": 1581407189000,
-                      "actor": "urn:li:corpuser:jdoe",
-                      "impersonator": null
-                    },
-                    "dataset": "urn:li:dataset:(urn:li:dataPlatform:hive,logging_events,PROD)",
-                    "type": "COPY"
-                  }
-                ]
-              }
-            },
-            {
-              "com.linkedin.pegasus2avro.schema.SchemaMetadata": {
-                "schemaName": "SampleHiveSchema",
-                "platform": "urn:li:dataPlatform:hive",
-                "version": 0,
-                "created": {
-                  "time": 1581407189000,
-                  "actor": "urn:li:corpuser:jdoe",
-                  "impersonator": null
-                },
-                "lastModified": {
-                  "time": 1581407189000,
-                  "actor": "urn:li:corpuser:jdoe",
-                  "impersonator": null
-                },
-                "deleted": null,
-                "dataset": null,
-                "cluster": null,
-                "hash": "",
-                "platformSchema": {
-                  "com.linkedin.pegasus2avro.schema.KafkaSchema": {
-                    "documentSchema": "{\"type\":\"record\",\"name\":\"SampleHiveSchema\",\"namespace\":\"com.linkedin.dataset\",\"doc\":\"Sample Hive dataset\",\"fields\":[{\"name\":\"field_foo\",\"type\":[\"string\"]},{\"name\":\"field_bar\",\"type\":[\"boolean\"]}]}"
-                  }
-                },
-                "fields": [
-                  {
-                    "fieldPath": "event_name",
-                    "jsonPath": null,
-                    "nullable": false,
-                    "description": {
-                      "string": "Name of your logging event"
-                    },
+              ]
+            }
+          },
+          {
```

Comment:
Do we need all the changes to this file?

Reply:
I will remove this, seems like it got autoformatted.